sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	# remove tests that require internet access to comply with Gentoo policy
 	sed -i -e '/CurlHttpClientTest.cpp/d' -e '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
 
+	# remove non-applicable warning caused by cmake-utils setting NDEBUG
```

Comment:
Yeah, sorry, it's an `#ifndef` in the .cpp file in question. I grepped the complete source tree and this line is the only place `NDEBUG` is used.